src/Response/SapiStreamEmitter.php

```diff
@@ -24,20 +24,13 @@ class SapiStreamEmitter implements EmitterInterface
      * body content via the output buffer.
      *
      * @param ResponseInterface $response
-     * @param null|int $maxBufferLevel Maximum output buffering level to unwrap.
      * @param int $maxBufferLength Maximum output buffering size for each iteration
      */
-    public function emit(ResponseInterface $response, $maxBufferLevel = null, $maxBufferLength = 8192)
+    public function emit(ResponseInterface $response, $maxBufferLength = 8192)
```

Comment:
Ironically, we changed the signature of this method to swap the `$maxBufferLength` and `$maxBufferLevel` arguments for 1.5.0, fixing an issue whereby 2 bytes at a time were streamed; had I resolved this issue prior to that release, we would not have needed to make that change, and then revert it here.